#!/Users/n01545735/MLHCBugs/conda/env/common/bin/python
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
